### PR TITLE
docs: fix docker commands

### PR DESCRIPTION
**Pull request by mshiznitzh**
Updated the docker documents to cover opening the port for webserver.

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -22,11 +22,12 @@ Here there are 3 different way to start the container and provide data persisten
 
 1. Replace `/dev/ttyACM0` with your serial device
 2. Add `-e TZ=Europe/Stockholm` to the `docker run` command to set the correct timezone in container
+3. Replace `3000:3000` with the Server Port of  the websocket
 
 ### Run using volumes
 
 ```bash
-docker run --rm -it -p 8091:8091 --device=/dev/ttyACM0 --mount source=zwavejs2mqtt,target=/usr/src/app/store zwavejs/zwavejs2mqtt:latest
+docker run --rm -it -p 8091:8091 -p3000:3000 --device=/dev/ttyACM0 --mount source=zwavejs2mqtt,target=/usr/src/app/store zwavejs/zwavejs2mqtt:latest
```
Comment (robertsLando) [inline docker/README.md:30]:
```suggestion
docker run --rm -it -p 8091:8091 -p 3000:3000 --device=/dev/ttyACM0 --mount source=zwavejs2mqtt,target=/usr/src/app/store zwavejs/zwavejs2mqtt:latest
```

File: docker/README.md [hunk 2]
```diff
@@ -35,7 +36,7 @@ Here we will store our data in the current path (`$(pwd)`) named `store`. You ca
 
 ```bash
 mkdir store
-docker run --rm -it -p 8091:8091 --device=/dev/ttyACM0 -v $(pwd)/store:/usr/src/app/store zwavejs/zwavejs2mqtt:latest
+docker run --rm -it -p 8091:8091 p3000:3000 --device=/dev/ttyACM0 -v $(pwd)/store:/usr/src/app/store zwavejs/zwavejs2mqtt:latest
```
Comment (robertsLando) [inline docker/README.md:39]:
```suggestion
docker run --rm -it -p 8091:8091 -p 3000:3000 --device=/dev/ttyACM0 -v $(pwd)/store:/usr/src/app/store zwavejs/zwavejs2mqtt:latest
```

File: docs/getting-started/quick-start.md [hunk 3]
```diff
@@ -22,6 +22,9 @@ docker-compose up
 > [!NOTE]
 > Replace `/dev/ttyACM0` with your serial device
 
+> [!NOTE]
+> Replace `p 3000:3000` with your Server Port setting for zwave-js websocket server
```
Comment (robertsLando) [inline docs/getting-started/quick-start.md:26]:
```suggestion
> Replace `-p 3000:3000` with your Server Port setting for zwave-js websocket server
```

**Review by robertsLando — CHANGES_REQUESTED** [review]

File: docker/README.md [hunk 4]
```diff
@@ -22,11 +22,12 @@ Here there are 3 different way to start the container and provide data persisten
 
 1. Replace `/dev/ttyACM0` with your serial device
 2. Add `-e TZ=Europe/Stockholm` to the `docker run` command to set the correct timezone in container
+3. Replace `3000:3000` with the Server Port of  the websocket
```
Comment (robertsLando) [inline docker/README.md:25]:
```suggestion
3. If you are using zwave-js WS server, replace `3000:3000` with the port choosen in settings
```

**Review by robertsLando — CHANGES_REQUESTED** [review]

File: docs/getting-started/quick-start.md [hunk 5]
```diff
@@ -22,6 +22,9 @@ docker-compose up
 > [!NOTE]
 > Replace `/dev/ttyACM0` with your serial device
 
+> [!NOTE]
+> Replace `-p 3000:3000` with your Server Port setting for zwave-js websocket server
```
Comment (robertsLando) [inline docs/getting-started/quick-start.md:26]: Same here

**Review by robertsLando — CHANGES_REQUESTED** [review]

**Review by robertsLando — DISMISSED** [review]

File: docs/getting-started/quick-start.md (hunk 5 above)
Comment (mshiznitzh) [reply docs/getting-started/quick-start.md]: Should be changed.  If I did it correctly

**Comment by robertsLando** [comment]
Please merge my pr

**Pull request merged**